cli/src/cli_util.rs

```diff
@@ -845,7 +845,7 @@ impl WorkspaceCommandHelper {
         self.workspace.working_copy()
     }
 
-    pub fn unsafe_start_working_copy_mutation(
+    pub fn unchecked_start_working_copy_mutation(
```

Comment:
> I don't have context for what was "unsafe" about this function to begin with, so thoughts on better names welcome.

It's about not checking for concurrent changes to the working copy since we loaded the working copy. The `unchecked_` sounds good, thanks.